include/AutomatableModel.h

```diff
@@ -210,6 +209,9 @@ class LMMS_EXPORT AutomatableModel : public Model, public JournallingObject
 	//! @brief Returns value applied with the inverse of this model's scale type
 	float inverseScaledValue( float value ) const;
 
+	//! @brief Returns the value at a buffer, if buffer not exists, returns fixed value
+	float valueAt(size_t index);
```

Comment:
@Rossmaxx It looks like you never implement this method. That's probably the cause of the build errors.

Reply:
I removed it for now.

Reply:
I did declare this `valueAt` function at line 478 but it's a part of `TypedAutomatableModel`, and wrote the definition too there.

Reply:
@JohannesLorenz I'll try reverting your solution for now.